lambda.js

```diff
@@ -18,6 +18,10 @@ const binaryMimeTypes = [
 const server = awsServerlessExpress.createServer(app, null, binaryMimeTypes)
 
 exports.handler = async (event, context) => {
-  console.log('request received for path: ', event.path);
-  return awsServerlessExpress.proxy(server, event, context, 'PROMISE').promise;
+  if (event.path) {
+    console.log('request received for path: ', event.path);
+    return awsServerlessExpress.proxy(server, event, context, 'PROMISE').promise;
+  } else { // assuming that this lambda method is triggered by EventBridge rule
+    // handle eventbridge rule
```

Comment:
@mahnoorfatiima 

assume that you are getting an object in `event.aqi` such that:

```javascript

console.log(event.aqi)
// cities : ["lahore", "islamabad", "peshawar"]
```

Reply:
- let's create a new controller layer for eventbridge i.e. create new folder `eventbridge/index.js` (CONTROLLER)
- create a new method inside index.js - `aqiPoster(event.aqi)`
- aqiPoster
    - fetch aqi of passed cities from AQI api
    - post it into slack using webhook

- we need `getAQI` in eventbridge controller too (MODEL)
    - key point: if two controllers need same code, it means that code should be business logic. so move into `modules/aqi.js` 

- time to post slack
   - we will use the following request to post data into our channel using `axios`:
```
curl -X POST -H 'Content-type: application/json' --data '{"text":"Hello, World from Muneeb K"}' https://hooks.slack.com/services/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
```
reference: https://api.slack.com/apps/AQK2B10HZ/incoming-webhooks